lib/geordi/commands/console.rb

```diff
@@ -7,8 +7,7 @@
 Selecting the server: `geordi console staging -s` shows a menu with all available
 servers. When passed a number, directly connects to the selected server.
 
-IRB flags defined in the global Geordi config file (`~/.config/geordi/global.yml`),
-are automatically passed to the irb console.
+IRB flags can be given as `irb_flags: '...' in the global Geordi config file (`~/.config/geordi/global.yml`).
```

Comment:
```suggestion
IRB flags can be given as `irb_flags: '...'` in the global Geordi config file (`~/.config/geordi/global.yml`).
```
Tip: review README changes (needs update!) in the Markdown preview.